src/any_agent/frameworks/any_agent.py

```diff
@@ -272,14 +272,12 @@ def serve(
             ```
 
         """
-        from any_agent.serving import A2AServingConfig, MCPServingConfig
+        from any_agent.serving import MCPServingConfig
 
         if isinstance(serving_config, MCPServingConfig):
             self._serve_mcp(serving_config)
-        elif isinstance(serving_config, A2AServingConfig):
-            self._serve_a2a(serving_config)
         else:
-            self._serve_a2a(None)
+            self._serve_a2a(serving_config)
 
     async def _serve_a2a_async(
         self, serving_config: A2AServingConfig | None
```

Comment:
```suggestion
        self, serving_config: "A2AServingConfig" | None
```

Import A2AServingConfig at the top of the file in a TYPE_CHECKING block